recipes/xtr/all/conanfile.py

```diff
@@ -17,19 +17,38 @@ class XtrConan(ConanFile):
         "fPIC": [True, False],
         "enable_exceptions": [True, False],
         "enable_lto": [True, False],
+        "enable_io_uring": [True, False],
+        "enable_io_uring_sqpoll": [True, False],
+        "sink_capacity_kb": ["default"] + list(map(lambda x : 2**x, range(2, 15))) # 4k to 16MB
```

Comment:
```suggestion
        "sink_capacity_kb": "ANY"
```

It's a special value which are accepts anything: https://docs.conan.io/en/latest/reference/conanfile/attributes.html#options